### ISSUE TITLE
Experience with sidekiq-repeat

### ISSUE THREAD

**Issue by maltoe** [FlavourSys/sidekiq-repeat#10]
Hi guys,

As a last message for today, I'd like to ask whether you had any sort of bad experience with this gem in the last couple years? I've used it in other projects as well and never had any reason to look at anything else, just wondered whether at FlavourSys something came up.

Thanks a lot! Have a wonderful weekend, and hope to see you soon for 🍺 

all the best

**Comment by maltoe** [comment]
Not sure if I follow: If you want the jobs to be scheduled in development, you need to manually require them? I thought the sidekiq server process would use Rails eager loading?

**Comment by dsander** [comment]
As far as I know Sidekiq switched to the Rails auto- and reloader with version 4. We have this hack in the `environment.rb` to ensure all jobs are loaded when the Sidekiq server boots.
```ruby
if Sidekiq.server? && Rails.env == 'development'
  # Trigger auto loading of all workers to ensure repeating jobs are registered
  Dir[Rails.root.join('app/workers/*.rb')].each { |f| File.basename(f, '.rb').camelcase.constantize }
end
```

**Comment by dsander** [comment]
Good idea. Maybe we could put that logic into a Railtie or in a `Sidekiq.configure_server` block in the gems code. We have been using our method for so long that I never really looked at alternatives.
